cylc/flow/network/schema.py

```diff
@@ -1429,7 +1429,7 @@ class Meta:
 
             For each task the final effect of all broadcasts to all namespaces
             is computed on the fly just prior to job submission.  The
-            `--cancel` and `--clear` options simply cancel (remove) active
+            "clear" mode cancels (removes) active
             broadcasts, they do not act directly on the final task-level
             result. Consequently, for example, you cannot broadcast to "all
```

Comment:
```suggestion
            broadcasts, it cannot act directly on the final task-level
            result. Consequently, for example, you cannot broadcast to "all
```

Reply:
The subject is "broadcasts" not "Broadcast" so the plural is correct?

Reply:
I thought the subject was "the clear mode"

Reply:
I think "they" referred to "the --cancel and --clear options", which you've replaced with a single "clear mode", so I think @MetRonnie is right.

Reply:
Ok.

On second reading this seems a bit dense so I've re-worked it.